src/ipyradiant/visualization/interactive_exploration.py

```diff
@@ -75,7 +78,8 @@
 
 class GetOutgoingPredicateObjects(SPARQLQueryFramer):
     """
-    Return all triples for non-Literal objects (and the optional object labels).
```

Comment:
This is not in master. I think it got added back during merge conflict resolution
https://github.com/jupyrdf/ipyradiant/blob/master/src/ipyradiant/visualization/interactive_exploration.py